TAF/utils/scripts/manual/deploy-edgex.sh

```diff
@@ -6,16 +6,21 @@ GLOBAL_VAR_DIR=${WORK_DIR}/TAF/config/global_variables.py
 
 if [ ${TARGET} ==  "KVM" ] || [ ${TARGET} ==  "kvm" ]
 then
-    cd ${WORK_DIR}/../edge-linux-test-pytest
+    cd ${PYTEST_PATH}
     echo "<<<<< Installing pytest related libraries >>>>>"
-    python3 -m pip install -r ${HOME}/edge-linux-test-pytest/requirements.txt
+    timeout 2m pip3 install -r requirements.txt --use-pep517
```

Comment:
@bhavikkothari-eaton is this hard coded time out if fine to have here?

@markabrezovec-eaton

Reply:
@rajatkumar-eaton yes, this is hard coded because any library will take approx. maximum 120 seconds to install.

Reply:
what if we get timeout after 120s also?

Reply:
I believe the Pune team has the most issues with timeouts.  I am find with a 2m timeout if that comfortably catches the issues.  Should we error handle the return code in these cases?  We shouldn't move forward if a necessary step fails.